webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -169,9 +171,7 @@ export default class CreateIssueModal extends PureComponent {
                 backdrop='static'
             >
                 <Modal.Header closeButton={true}>
-                    <Modal.Title>
-                        {'Create GitHub Issue'}
-                    </Modal.Title>
+                    <Modal.Title>{'Create GitHub Issue'}</Modal.Title>
```

Comment:
I like all of the style improvements in this PR, but I personally think this one is more readable with the content on its own line as it was before